src/style/typography/h2.tsx

```diff
@@ -1,15 +1,20 @@
 import { colors } from '../colors';
 import { components } from './typography';
 
-export const H2 = ({ children }: { children: React.ReactNode }) => {
+interface H2Props {
+  children: React.ReactNode;
+  color?: "Xdark" | "dark";
+}
+
+export const H2 = ({ children, color }: H2Props) => {
   return (
     <h2
       style={{
         fontFamily: components.family.primary,
         fontSize: components.fontSize.xxLarge,
         fontWeight: components.fontWeight.regular,
         lineHeight: components.lineHeight.large,
-        color: colors.brand.medium,
+        color: color === "Xdark" ? colors.neutral.neutralXDark : colors.neutral.neutralMedium,
```

Comment:
se a cor for `dark`, era espeardo que recebesse um `neutralDark`, não? poderia usar o nome `medium`, ao inves do `dark` entao, o que acha?